views/components/sortableTable.njk

```diff
@@ -0,0 +1,171 @@
+{#
+  Sortable Table Component
```

Comment:
Observation: As noted when given a demo of the changes, it would be good to have this file name `sortableTable Component`  so that we have the consistent nomenclature.